mitosheet/mitosheet/public/v2/__init__.py

```diff
@@ -0,0 +1,8 @@
+# V2 is the same as V1, but it has a new module, for testing reasons
```

Comment:
Now that we've tested this and are creating v3, do we still want to keep v2 if it just for testing?

Reply:
Yeah, let's keep it!